backends/magma/ceed-magma.c

```diff
@@ -682,7 +682,7 @@ static int CeedBasisDestroy_Magma(CeedBasis basis) {
 static int CeedBasisCreateTensorH1_Magma(Ceed ceed, CeedInt dim, CeedInt P1d,
     CeedInt Q1d, const CeedScalar *interp1d,
     const CeedScalar *grad1d,
-    const CeedScalar *qref1d,
+    const CeedScalar *qmagma1d,
```

Comment:
I don't think this is correct. "qref" is referring to the location of quadrature points on the reference element, not the `ref` backend.

Reply:
Fixed